demo-opensuse/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM opensuse
+MAINTAINER Shikher Verma
+
+RUN cd home/
+RUN zypper --non-interactive install git python-pip gcc python-devel libffi-devel libopenssl-devel vim build sudo osc wget
+RUN zypper --non-interactive install -t pattern devel_C_C++
+RUN (cd home/ && git clone https://github.com/in-toto/in-toto.git)
+RUN (cd home/in-toto/ && pip install --upgrade pip setuptools virtualenv)
+RUN (cd home/in-toto/ && virtualenv -p /usr/bin/python2.7 in-toto)
+RUN (cd home/in-toto && source in-toto/bin/activate && pip install -r requirements.txt)
+RUN (cd home/in-toto && source in-toto/bin/activate && pip install -e .)
```

Comment:
I wonder if we need a venv on the container. I have a feeling that we can install these system-wide given that the container is just serving the purpose of running in-toto commands. It also let's us drop the source venv.